app/gui/view/component-browser/searcher-list-panel/src/lib.rs

```diff
@@ -760,6 +866,26 @@ impl EntryId {
     }
 }
 
+/// The selected part of the component group and the identifier of this group.
+///
```

Comment:
If this structure is a "component group AND the identifier of this group" then it should contain BOTH information, but it contains only one of them.

Reply:
Slightly refactored this struct. It contains group id in both variants, it just wasn't obvious enough.